helpdesk-maker-checker-service/src/main/resources/ActionMessage.xml

```diff
@@ -0,0 +1,9 @@
+<messageML>
+    <div class="entity" data-entity-id="makerchecker">
+        <card class="barStyle">
+            <header>
+                ${entity['makerchecker'].checker.displayName} ${entity['makerchecker'].state} this message. It has been delivered to the client(s).
```

Comment:
When the state is approved, this phrase It has been delivered to the client(s). are correct, but, if denied, it should be
It has not been delivered to the client(s).

Reply:
Done.